packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -418,8 +419,23 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
       input.eventTypeId
     }`
   );
+
+  // The effectiveMinimumBookingNotice just makes sure we don't discard slots too early, so we can later decide
+  // which minimumBookingNotice to apply based on if the slot has any attendees or not.
+  let seatsMinimumBookingNoticeActive = false;
+  let effectiveMinimumBookingNotice = eventType.minimumBookingNotice;
+  // not using ternary's here to help TS narrow down the type of effectiveMinimumBookingNotice correctly
+  if (
+    eventType.seatsPerTimeSlot &&
+    eventType.seatsMinimumBookingNotice !== null &&
```

Comment:
```suggestion
    !isNaN(eventType.seatsMinimumBookingNotice) &&
```

Possibly for clarity; `undefined` is used below as `undefined < {minimumBookingNoticeValue} which will always be false, but it's quite unclear this works this way.